java/arcs/android/storage/handle/AndroidHandleManager.kt

```diff
@@ -57,7 +59,9 @@ fun AndroidHandleManager(
          * Create an [ActivationFactory] that will create [ServiceStore] instances that can manage
          * singleton [RawEntities]
          */
-        override fun singletonFactory() = SingletonServiceStoreFactory<RawEntity>(
+        override fun <T : Referencable> singletonFactory(
+            typeClass: KClass<T>
```

Comment:
What's this `typeClass` parameter used for?

Reply:
Oh hey, whoops. This is a vestige of stuff i was working on.. will remove.

Reply:
I think I like what you were working on... :-D